bench/workgen/workgen.cpp

```diff
@@ -2348,7 +2348,7 @@ Operation::kv_gen(
     TableOperationInternal *internal = static_cast<TableOperationInternal *>(_internal);
 
     uint_t size = iskey ? internal->_keysize : internal->_valuesize;
```

Comment:
Should this one be updated too?

Reply:
I don't think so because they are the size which should be much smaller vs the max which can be large.